Solidner/Views/Onboarding/FoodStageSelectView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  FoodStageSelectView.swift
+//  Solidner
+//
+//  Created by 황지우2 on 2023/11/13.
+//
+
+import SwiftUI
+
+struct FoodStageSelectView: View {
+    let guidingBubbleColor = Color(#colorLiteral(red: 0.7149112821, green: 0.7669619918, blue: 0.6710264087, alpha: 1))
+    let dividerBubbleColor = Color(#colorLiteral(red: 0.8509803922, green: 0.8509803922, blue: 0.8509803922, alpha: 1)).opacity(0.2)
+    let downChevronColor = Color(#colorLiteral(red: 0.7176470588, green: 0.7176470588, blue: 0.7176470588, alpha: 1))
+    let foodStagePickerColor = Color(#colorLiteral(red: 0.5019607843, green: 0.5568627451, blue: 0.6745098039, alpha: 1)).opacity(0.1)
+    @State var dayNumber = 0.0
```

Comment:
```suggestion
    @State private var dayNumber = 0.0
```
p3;
@State 는 private 으로 선언해주세요!